vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -846,6 +864,26 @@ public Registration addInvalidChangeListener(
         return addListener(InvalidChangeEvent.class, listener);
     }
 
+    private String getBadInputErrorMessage() {
+        return Optional.ofNullable(i18n)
+                .map(DatePickerI18n::getBadInputErrorMessage).orElse("");
+    }
+
+    private String getRequiredErrorMessage() {
+        return Optional.ofNullable(i18n)
+                .map(DatePickerI18n::getRequiredErrorMessage).orElse("");
+    }
+
+    private String getMinErrorMessage() {
+        return Optional.ofNullable(i18n).map(DatePickerI18n::getMinErrorMessage)
+                .orElse("");
+    }
+
+    private String getMaxErrorMessage() {
+        return Optional.ofNullable(i18n).map(DatePickerI18n::getMaxErrorMessage)
+                .orElse("");
+    }
```

Comment:
Could use the same approach suggested in https://github.com/vaadin/flow-components/pull/6365#discussion_r1675358980

Reply:
Thanks, updated.